oiv/oiv.vcxproj

```diff
@@ -94,15 +94,15 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>OIV_NO_CLIENT_BUILD;OGRE_SOURCE_DIR="$(OGRESOURCE)";%(PreprocessorDefinitions);WIN32_LEAN_AND_MEAN;VC_EXTRALEAN;VC_EXTRALEAN</PreprocessorDefinitions>
-      <AdditionalIncludeDirectories>$(SolutionDir)\LLUtils\Include;$(SolutionDir)\ImageCodec\Image\Include;$(SolutionDir)\ImageCodec\ImageLoader\Include;$(SolutionDir)\ImageUtil\Include;$(ProjectDir);E:\Downloads\freetype-2.3.5-1-bin\include\;E:\Downloads\freetype-2.3.5-1-bin\include\freetype2\;%(AdditionalIncludeDirectories);$(SolutionDir)\oiv</AdditionalIncludeDirectories>
```

Comment:
Using 'OIVRootDir'  is still not good enough for the external dependencies.
I suggest creating a global macro for every dependency using visual studio property sheets and then use the 'AdditionalIncludeDirectories' e.g.:
`
<AdditionalIncludeDirectories>$(OPENGL_DIR)\Include;$(FREETYPE_DIR)\Include\FreeType2</AdditionalIncludeDirectories>`
The same could be done also to the ImageCodec
Later on it would be quite easy to generate the property sheet file.

Reply:
I agree. Will try to submit a fix for this issue in the next commit.